autopart-encrypted-1.ks.in

```diff
@@ -18,17 +18,26 @@ shutdown
 %end
 
 %post
-# Check if the type of /dev/vda2 is crypto_LUKS.
-crypted="/dev/vda2"
-type="$(blkid -o value -s TYPE ${crypted})"
+# Log the created partitioning.
+lsblk
```

Comment:
Just making sure, do we want to log the partitioning only in autopart-encrypted-1 ?

Reply:
I have got lazy.